addons/mass_mailing/static/src/snippets/s_social_media_mass_mailing/options.js

```diff
@@ -0,0 +1,109 @@
+odoo.define('@web_editor/src/snippets/s_social_media_mass_mailing/options', function (require) {
```

Comment:
Never use odoo.define anymore, just use the new (well it's not new anymore) system to declare odoo JS module